lib/helpers/android.ts

```diff
@@ -814,45 +814,51 @@ const AndroidHelpers: AndroidHelpers = {
         `${SETTINGS_HELPER_PKG_ID} is already running. ` +
           `There is no need to reset its permissions.`
       );
-      return;
```

Comment:
I still don't understand why the `return` has been removed here. The below adjustments will only be done if the process is not running yes, so what is the point in restarting it afterwards? It might also take extra time on the session startup.